src/Compilers/Core/Portable/RealParser.cs

```diff
@@ -0,0 +1,806 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Numerics;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Microsoft.CodeAnalysis
+{
+#if OS_UNIX
+
+    // TODO: https://github.com/dotnet/roslyn/issues/5870
+    internal static class RealParser
+    {
+        public static bool TryParseDouble(string s, out double d)
+        {
+            return double.TryParse(s, out d);
```

Comment:
@gafter bah, I didn't think about that.  I just ran the build on my machine until I confirmed the bootstrap process passed.  If any tests fail on Unix I'll disable them for Unix runs only (referencing the tracking bug).